source/Octopus.Cli/Infrastructure/Options.cs

```diff
@@ -663,14 +665,24 @@ protected override void OnParseComplete(OptionContext c)
             }
         }
 
-        public OptionSet Add<T>(string prototype, Action<T> action)
-        {
-            return Add(prototype, null, action);
-        }
-
-        public OptionSet Add<T>(string prototype, string description, Action<T> action, bool sensitive = false)
-        {
-            return Add(new ActionOption<T>(prototype, description, action, sensitive));
+        /// <summary>
+        /// </summary>
+        /// <param name="prototype">The option name.
+        /// Can be used to specify aliases by separating with a | character.
```

Comment:
Nice 👍

I found these special characters a little confusing when I first used them, because I wasn't sure how significant they were and what they meant. This will smooth the path a little for new engineers.